business_requirement_deliverable_project/models/business.py

```diff
@@ -44,6 +44,24 @@ def generate_projects_wizard(self):
         compute='_compute_planned_hour'
     )
 
+    all_project_generated = fields.Boolean(compute='compute_project_generate',
+                                           string='All Project Generated')
+
+    @api.depends('business_requirement_ids',
+                 'business_requirement_ids.linked_project')
+    def compute_project_generate(self):
```

Comment:
Field name is `all_project_generated`
then this function should be named `compute_all_project_generated`